changelogs/current.yaml

```diff
@@ -49,6 +49,10 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: buffer
+  change: |
+    Fixed a bug the memory will be released if it is coming from BufferFragment, when moving the slices between
+    the buffers and clear the drain tracker at the same time.
```

Comment:
```suggestion
    Fixed a bug where the memory will be released if it is coming from ``BufferFragment`` when moving the slices between
    the buffers. This clear the drain tracker at the same time.
```

Reply:
not sure iiuc but i think the grammar needs a bit of a rejig

Reply:
@phlax thanks!, let me fix it

Reply:
I keep use the `and` in `when moving the slices between the buffers and clear the drain tracker at the same time.`, since I want to say this bug only trigger with this code:
`dst.move(src, res, /*reset_drain_trackers_and_accounting=*/true);`

`reset_drain_trackers_and_accounting` need to be true.